src/views/About.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="tos-container">
```

Comment:
tos ?
is just borrowed from terms of service page?

Reply:
yes. it seemed tos styles were also used in the privacy page, so I went on and used it on the about page. therefore I thought we could have layout components (my Nuxt background wants layouts!!! haha)